src/test/scala/com/microsoft/hyperspace/index/E2EHyperspaceRulesTests.scala

```diff
@@ -407,7 +407,9 @@ class E2EHyperspaceRulesTests extends HyperspaceSuite with SQLHelper {
   }
 
   private def getIndexFilesPath(indexName: String): Path = {
-    new Path(systemPath, s"$indexName/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=0")
+    val fs = systemPath.getFileSystem(new Configuration)
```

Comment:
Can we create a utils function that takes in `Path` and returns absolute path if it's not already absolute path?

Reply:
thanks @imback82 , done